blas/testblas/common.go

```diff
@@ -276,6 +276,28 @@ func zsame(x, y []complex128) bool {
 	return true
 }
```

Comment:
This probably should have been committed before the tests, but github will never render it that way now.

Reply:
I reordered the commits, so maybe now they are rendered as you wished?

Reply:
It will make the commits all pass which is the important thing. Github for some reason renders commits by creation time, not graph order.